src/function/export/export_parquet_function.cpp

```diff
@@ -115,8 +121,9 @@ static void finalizeFunc(ExportFuncSharedState& sharedState) {
 function_set ExportParquetFunction::getFunctionSet() {
     function_set functionSet;
     auto exportFunc = std::make_unique<ExportFunction>(name);
-    exportFunc->initLocal = initLocalState;
-    exportFunc->initShared = initSharedState;
+    exportFunc->initLocalState = initLocalStateFunc;
+    exportFunc->createSharedState = createSharedStateFunc;
+    exportFunc->initSharedState = initSharedStateFunc;
```

Comment:
I thought the normal design is to merge  createSharedState inside initSharedState just like the previous design. I believe we did this in multiple places not just for export